src/views/Item/Item.jsx

```
@@ -353,6 +494,92 @@ const Item = () => {
+                                borderRadius: "10px",
+                                padding: "5px",
+                                backgroundColor: "#ffffff",
+                            }}
+                            className="hover:bg-gray-100 font-semibold"
+                            onClick={() => {
+                                setIsFilterApplied(true);
+                                fetchFilteredData();
+                                setFilterModal(false);
+                                setLoading(true);
+                            }}
+                        >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)